tests/test_buildrunner_files.py

```diff
@@ -186,14 +185,11 @@ def test_buildrunner_arm_dir(test_dir: str, file_name, args, exit_code):
     _test_buildrunner_file(test_dir, file_name, args, exit_code)
 
 
-@pytest.mark.serial
-@pytest.mark.skipif(
-    "arm64" not in platform.uname().machine,
-    reason="This test should only be run on arm64 architecture",
-)
+@pytest.mark.flaky(reruns=2, reruns_delay=1)
 @pytest.mark.parametrize(
     "test_dir, file_name, args, exit_code",
-    _get_test_runs(test_dir=f"{TEST_DIR}/test-files/arm-arch", serial_tests=True),
+    _get_test_runs(test_dir=f"{TEST_DIR}/test-files/scan", serial_tests=False),
 )
-def test_serial_buildrunner_arm_dir(test_dir: str, file_name, args, exit_code):
```

Comment:
There are no files for `test_serial_buildrunner_arm_dir` to run. So I removed this test.